From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bastiaan van der Plaat <bastiaan.v.d.plaat@gmail.com>
Date: Sat, 10 Oct 2026 12:35:54 +0200
Subject: [PATCH] Include stddef.h for size_t

Headers like ogc/system.h use size_t without including stddef.h, this
worked because newlib's stdint.h brings it along, clang's doesn't.
---
 gc/gctypes.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gc/gctypes.h b/gc/gctypes.h
index 0e53704..41fbf9d 100644
--- a/gc/gctypes.h
+++ b/gc/gctypes.h
@@ -7,6 +7,7 @@
 */ 
 
 #include <stdbool.h>
+#include <stddef.h>
 #include <stdint.h>
 
 #ifdef __cplusplus
